app/page.tsx

```diff
@@ -1,54 +1,58 @@
+          <Link href="/how_it_works">How It Works</Link>
+          <Link href="/why_minted">Why Minted</Link>
+        </div>
+        {/* Log In and Dashboard Button */}
+        <div className="flex flex-row flex-1 text-heading justify-end items-center">
+          {email && <div className="2xl:pr-10 pr-6 text-xl font-medium">Hey, {email}!</div>}
+          {email ?
+          <Link href="/dashboard" className="2xl:mr-10 mr-6 2xl:h-16 h-12 2xl:w-40 w-32 border-4 border-darkmaroon border-solid rounded-full text-darkmaroon 2xl:text-xl text-lg font-semibold flex justify-center items-center hover:bg-darkmaroon hover:text-primary transition duration-300">Dashboard</Link>
+          :
+          <Link href="/login" className="2xl:mr-10 mr-6 2xl:h-16 h-12 2xl:w-40 w-32 border-4 border-darkmaroon border-solid rounded-full text-darkmaroon 2xl:text-xl text-lg font-semibold flex justify-center items-center hover:bg-darkmaroon hover:text-primary transition duration-300">Log In</Link>
+          }
         </div>
-      </nav>
-
-      <div className="animate-in flex-1 flex flex-col gap-20 opacity-0 max-w-4xl px-3">
-        <Header />
-        <main className="flex-1 flex flex-col gap-6">
-          <h2 className="font-bold text-4xl mb-4">Next steps</h2>
-          {isSupabaseConnected ? <SignUpUserSteps /> : <ConnectSupabaseSteps />}
-        </main>
       </div>
 
-      <footer className="w-full border-t border-t-foreground/10 p-8 flex justify-center text-center text-xs">
-        <p>
-          Powered by{" "}
-          <a
-            href="https://supabase.com/?utm_source=create-next-app&utm_medium=template&utm_term=nextjs"
-            target="_blank"
-            className="font-bold hover:underline"
-            rel="noreferrer"
-          >
-            Supabase
-          </a>
-        </p>
-      </footer>
+      {/* Center Text */}
+      <div className="relative space-y-8 flex flex-col text-center">
+        <div className="space-y-8 2xl:text-8xl text-7xl text-darkmaroon font-semibold">
+          <div>Built for Non-Profits</div>
+          <div>Built for Change</div>
+        </div>
+        <div className="flex justify-center">
+          <div className="w-1/2 2xl:text-3xl text-2xl text-heading font-medium">Find opportunities to fund projects or start funding non-profits. We’re here to boost projects.</div>
+        </div>
+        {/* Bottom Buttons */}
+        <div className="space-x-10 flex flex-row justify-center 2xl:text-2xl text-xl font-medium">
+          <Link href={email ? "/dashboard/browse/opengrants" : "/login"} className="2xl:h-12 h-10 2xl:w-56 w-44 bg-darkmaroon rounded-full text-primary flex justify-center items-center hover:brightness-125">Find Funding {'>'}</Link>
+          <Link href={email ? "/dashboard/organization/grants" : "/login"} className="2xl:h-12 h-10 2xl:w-56 w-44 rounded-full text-darkmaroon flex justify-center items-center hover:brightness-125">Fund now {'>'}</Link>
```

Comment:
Users could be part of a non-profit so they shouldn't have access to grants when we redirect them using this button. We need to add a check in the grants page to redirect non-profit users out of it when they try clicking this button. Could make a GH issue so we don't forget.

Reply:
Would the GH issue call for checking on entering the grants page?

Reply:
Yea. Technically we should be having a check on every dashboard page to make sure ppl in other org types shouldn't have access to the other org type pages but since we're giving all users the direct link here we'll have to make sure we implement that check under the /dashboard/organization/grants page.

Reply:
Yeah, I think it would be better to add a check on every dashboard page instead of changing the button link since users can still technically enter the page.